index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CALCULATOR</title>
+  <link rel="stylesheet" href="styles\main.css">
+  <script src="scripts\main.js" defer></script>
+  <link href="https://fonts.googleapis.com/css2?family=Gemunu+Libre:wght@300;400;500;600;700;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="calculator">
+    <div class="calc-container">
+      <div id="display">
+        <div class="characterdisplay"></div>
+        <div class="resultdisplay"></div>
+      </div>
+      <div class="character">
+        <div class="group first">
+          <button type="button" class="clear" value="clear">AC</button>
+          <button type="button" id="sign" class="sign" value="sign">+/-</button>
+          <button type="button" id="percent" class="percent" value="percentage">%</button>
+          <button type="button" id="divi" class="operator division" value="divide">/</button>
+        </div>
+        <div class="group second">
+          <button type="button" class="operand seven" value="7">7</button>
+          <button type="button" class="operand eight" value="8">8</button>
+          <button type="button" id="nine" class="operand nine" value="9">9</button>
+          <button type="button" id="mul" class="operator multiply" value="multiply">*</button>
+        </div>
+        <div class="group third">
+          <button type="button" id="four" class="operand four" value="4">4</button>
+          <button type="button" id="five" class="operand five" value="5">5</button>
+          <button type="button" id="six" class="operand six" value="6">6</button>
+          <button type="button" id="sub" class="operator subtraction" value="subtract">-</button>
+        </div>
+        <div class="group fourth">
+          <button type="button" id="one" class="operand one" value="1">1</button>
+          <button type="button" id="two" class="operand two" value="2">2</button>
+          <button type="button" id="three" class="operand three" value="3">3</button>
+          <button type="button" id="add" class="operator addition" value="add">+</button>
+        </div>
+        <div class="group fifth">
+          <button type="button" id="zero" class="operand zero" value="0">0</button>
+          <button type="button" class="operand dot" value="dot">.</button>
+          <button type="button" id="delete">DEL</button>
+          <button type="button" class="operator equalto" value="equalto">=</button>
+          
+        </div>
+      </div>
+    </div>
+  </div>
```

Comment:
You've developed the entire structure of your app using the `div` container tag. This is a bad practice. You should consider using [semantic HTML ](https://www.w3schools.com/html/html5_semantic_elements.asp)because of [these advantages](https://medium.com/adalab/the-importance-of-semantic-html-78e74fb75ff0).

Note: this does not mean you should abuse the semantic HTML tags. You should only use them where necessary.